### PR TITLE
[Feature-7785][UI Next]Add a new page for the module of next ui to manage the task group option.

### LINKED ISSUE #7785: [Feature][UI Next] Resource.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- [x] File Manage
- [x] Resource Manage
- [ ] Function Manage
- [x] Task Group Option
- [x] Task Group Queue

### Use case

_No response_

### Related issues

[#7332](https://github.com/apache/dolphinscheduler/issues/7332)

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by calvinjiang**
## Purpose of the pull request

This PR will close #7785 .

## Verify this pull request
This change added tests and can be verified as follows:
  - *Manually verified the change by testing locally.*
 image

image

image

### CODE REVIEW

File: dolphinscheduler-ui-next/.env.development [hunk 1]
```diff
@@ -15,4 +15,4 @@
 
 NODE_ENV = development
 
-VITE_APP_WEB_URL = ''
+VITE_APP_WEB_URL = 'http://localhost:12345'
```
Comment (labbomb) [inline dolphinscheduler-ui-next/.env.development]: Please delete this address

**Review by labbomb — COMMENTED** [review]
@songjianet Basically, I think there's no problem. You can take a look

**Review by labbomb — DISMISSED** [review]
LGTM

**Review by songjianet — DISMISSED** [review]
LGTM

**Comment by songjianet** [comment]
Please merge the latest code in the current branch through 'rebase'. @calvinjiang

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**